debug_utils/sierra-emu/src/utils.rs

```diff
@@ -0,0 +1,56 @@
+use cairo_lang_sierra::{
+    extensions::core::{CoreLibfunc, CoreType, CoreTypeConcrete},
+    program_registry::ProgramRegistry,
+};
+use num_bigint::BigInt;
+use num_traits::ToPrimitive;
+
+use crate::Value;
+
+/// Receives a vector of values, filters any which is non numeric and returns a `Vec<BigInt>`
+/// Useful when a binary operation takes generic values (like with bounded ints).
+pub fn get_numberic_args_as_bigints(args: Vec<Value>) -> Vec<BigInt> {
```

Comment:
We can receive a slice to avoid cloning the vector calling this function.
```suggestion
pub fn get_numberic_args_as_bigints(args: &[Value]) -> Vec<BigInt> {
```

Reply:
Done